### PR TITLE
CATTY-108 Convert FormulaFunctionsTest from ObjectiveC to Swift.

### PR DESCRIPTION
**Pull request by roelac11**
Convert FormulaFunctionsTest from ObjectiveC to Swift.

### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Catty/blob/develop/README.md) and [wiki pages](https://github.com/Catrobat/Catroid/wiki/) of this repository.

- [x] Include the name of the Jira ticket in the PR’s title
- [x] Include a summary of the changes plus the relevant context
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no compiler or linter warnings
- [x] Perform a self-review of the changes
- [x] Verify to commit no other files than the intentionally changed ones
- [x] Include reasonable and readable tests verifying the added or changed behavior
- [x] Confirm that new and existing unit tests pass locally
- [x] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [x] Stick to the project’s git workflow (rebase and squash your commits)
- [x] Verify that your changes do not have any conflicts with the base branch
- [x] After the PR, verify that all CI checks have passed
- [x] Post a message in the *#ios-general* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

File: src/CattyTests/Formula/FormulaFunctionsTest.swift [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+
+import XCTest
+
+@testable import Pocket_Code
+
+final class FormulaFunctionsTest: XCTestCase {
+    var formulaManager: FormulaManager!
+    var spriteObject: SpriteObject!
+
+    let EPSILON = 0.001
```
Comment (m-herold) [inline src/CattyTests/Formula/FormulaFunctionsTest.swift]: Please use our extension instead (Double.epsilon).

File: src/CattyTests/Formula/FormulaFunctionsTest.swift [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+        let screenSize = Util.screenSize(true)
+        formulaManager = FormulaManager(sceneSize: screenSize)
+        spriteObject = SpriteObject()
+    }
+
+    func getFormulaForFunction(tag: String, leftValue: String, rightValue: String?) -> Formula {
+        let leftElement = FormulaElement(elementType: ElementType.NUMBER, value: leftValue, leftChild: nil, rightChild: nil, parent: nil)
+        var rightElement: FormulaElement?
+
+        if rightValue != nil {
```
Comment (m-herold) [inline src/CattyTests/Formula/FormulaFunctionsTest.swift]: Just to let you know: please prefer "if let" in order to remove to force-unwrap.

Comment (GatCode) [reply src/CattyTests/Formula/FormulaFunctionsTest.swift]: Yes please update this!

**Comment by m-herold** [comment]
Please also resolve merge conflicts.

**Comment by m-herold** [comment]
Jenkins: retest this please

**Pull request merged**